Adressing Usama's review (issue #64) (#70)
ietf-rats-wg/draft-ietf-rats-reference-interaction-models: draft-ietf-rats-reference-interaction-models.md

If an entity takes on two or more different roles, the functions they provide typically reside in isolated environments that are components of the same entity.
Examples of such isolated environments include a Trusted Execution Environment (TEE), Baseboard Management Controllers (BMCs), as well as other physical or logical protected/isolated/shielded Computing Environments (e.g., embedded Secure Elements (eSE) or Trusted Platform Modules (TPM)).

## Boot Time Integrity

> Boot time integrity refers to the trustworthiness of the platform during its boot sequence, typically covering firmware, BIOS/UEFI, initial bootloaders, and core operating system components up until a stable runtime environment is reached.

Author: ```suggestion
Boot time integrity refers to the trustworthiness of the platform during its boot sequence, typically covering firmware, BIOS/UEFI, initial bootloaders, and core operating system components up until a stable runtime environment is reached (typically via layered attestation).
```

## Challenge/Response Remote Attestation {#challenge-response}

> Note: In the following diagrams, a leading `?` indicates that a parameter is optional.

Author: ```suggestion
Note: In the following diagrams, a leading `?` indicates that an information element is optional.
```

     |                                       attestationResult <= |
     |                                                            |
~~~~
{: #fig-challenge-response title="Figure 1: Challenge/Response Remote Attestation"}

The Attester boots up and thereby produces Claims about its boot state and its operational state during runtime (cf. {{terminology}}), e.g., loaded applications, configurations, and environment variables.
Event Logs may accompany the produced Claims and provide an event trail of security-critical events in the system. Claims are produced by all Attesting Environments of an Attester system.

The Challenge/Response remote attestation procedure is typically initiated by the Verifier by sending a remote attestation request to the Attester.

> Alternative initiation flows, e.g., via an intermediary or through pre-configured requests, are out of scope for this document.

Author: ```suggestion
Alternative initiation flows, e.g., via an intermediary or through pre-configured requests (e.g., Call-Home procedures or trusted trigger events from Relying Parties), are out of scope for this document.
```
